src/js/colors/colors.ts

```diff
@@ -1,199 +1,14 @@
-      alternative: '#F2F4F6',
-    },
-    text: {
-      default: '#24272A',
-      alternative: '#535A61',
-      muted: '#BBC0C5',
-    },
-    icon: {
-      default: '#24272A',
-      alternative: '#6A737D',
-      muted: '#BBC0C5',
-    },
-    border: {
-      default: '#BBC0C5',
-      muted: '#D6D9DC',
-    },
-    overlay: {
-      default: '#00000099',
-      inverse: '#FCFCFC',
-      alternative: '#000000CC',
-    },
-    primary: {
-      default: '#037DD6',
-      alternative: '#0260A4',
-      muted: '#037DD619',
-      inverse: '#FCFCFC',
-      disabled: '#037DD680',
-    },
-    secondary: {
-      default: '#F66A0A',
-      alternative: '#C65507',
-      muted: '#F66A0A19',
-      inverse: '#FCFCFC',
-      disabled: '#F66A0A80',
-    },
-    error: {
-      default: '#D73A49',
-      alternative: '#B92534',
-      muted: '#D73A4919',
-      inverse: '#FCFCFC',
-      disabled: '#D73A4980',
-    },
-    warning: {
-      default: '#F66A0A',
-      alternative: '#FFC70A',
-      muted: '#FFD33D19',
-      inverse: '#FCFCFC',
-      disabled: '#FFD33D80',
-    },
-    success: {
-      default: '#28A745',
-      alternative: '#1E7E34',
-      muted: '#28A74519',
-      inverse: '#FCFCFC',
-      disabled: '#28A74580',
-    },
-    info: {
-      default: '#037DD6',
-      alternative: '#0260A4',
-      muted: '#037DD619',
-      inverse: '#FCFCFC',
-      disabled: '#037DD680',
-    },
+    ...lightTheme.colors,
   },
   dark: {
-    background: {
-      default: '#24272A',
-      alternative: '#141618',
-    },
-    text: {
-      default: '#FFFFFF',
-      alternative: '#D6D9DC',
-      muted: '#9FA6AE',
-    },
-    icon: {
-      default: '#FFFFFF',
-      alternative: '#BBC0C5',
-      muted: '#9FA6AE',
-    },
-    border: {
-      default: '#848C96',
-      muted: '#3B4046',
-    },
-    overlay: {
-      default: '#00000099',
-      inverse: '#FCFCFC',
-      alternative: '#000000CC',
-    },
-    primary: {
-      default: '#1098FC',
-      alternative: '#43AEFC',
-      muted: '#1098FC26',
-      inverse: '#FCFCFC',
-      disabled: '#1098FC80',
-    },
-    secondary: {
-      default: '#F8883B',
-      alternative: '#FAA66C',
-      muted: '#F8883B26',
-      inverse: '#FCFCFC',
-      disabled: '#F8883B80',
-    },
-    error: {
-      default: '#D73A49',
-      alternative: '#E06470',
-      muted: '#D73A4926',
-      inverse: '#FCFCFC',
-      disabled: '#D73A4980',
-    },
-    warning: {
-      default: '#FFD33D',
-      alternative: '#FFDF70',
-      muted: '#FFD33D26',
-      inverse: '#141618',
-      disabled: '#FFD33D80',
-    },
-    success: {
-      default: '#28A745',
-      alternative: '#5DD879',
-      muted: '#28A74526',
-      inverse: '#FCFCFC',
-      disabled: '#28A74580',
-    },
-    info: {
-      default: '#1098FC',
-      alternative: '#43AEFC',
-      muted: '#1098FC26',
-      inverse: '#FCFCFC',
-      disabled: '#037DD680',
-    },
+    ...darkTheme.colors,
   },
 };
```

Comment:
Updating deprecated colors object to use the new lightTheme and darkTheme colors so we don't have to update colors in 2 places. I think I will probably add the brand colors as an internal global object in another PR